utils/frame-utils/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "frame-utils"
```

Comment:
yeah i agree, frame_support is a better place for it.

Reply:
ended up moving it to `frame_system::extras` because `frame_system` depends on `frame_support` whereas `frame_utils` depends on `frame_system` and so, therefore, can't be in `frame_support`